### ISSUE TITLE
Issue with compiling bezierCurve

### ISSUE THREAD

**Issue by ItzRabbs** [Xetera/ghost-cursor#41]
I'm getting an issue with running tsc. See below. 

>node_modules/ghost-cursor/lib/math.d.ts:19:107 - error TS2304: Cannot find name 'Bezier'.
>
>19 export declare const bezierCurve: (start: Vector, finish: Vector, overrideSpread?: number | undefined) => Bezier;

**Comment by bcolthurst-raices** [comment]
It seems that in /src/math.ts

If I change
import Bezier from 'bezier-js'

to
import { Bezier } from 'bezier-js'

it changes the lib/math.d.ts last line from:
export declare const bezierCurve: (start: Vector, finish: Vector, overrideSpread?: number | undefined) => Bezier;

to
export declare const bezierCurve: (start: Vector, finish: Vector, overrideSpread?: number | undefined) => any;

Which could be a fix.

**Comment by Niek** [comment]
Strange, I cannot reproduce this. Is this on the latest master branch? Did you run `yarn` to build?

**Comment by bcolthurst-raices** [comment]
we have a workaround: 

in our project's package.json,
changing it from     "ghost-cursor": "^1.1.7", to     "ghost-cursor": "1.1.6" 
seems to fix it.

some clues about our Dockerfile:
it's FROM node:14
using
RUN npm install -g typescript@4.3.5

Now, when I was just cloning the ghost-cursor repo, most recent version, and doing the yarn/husky, npm run prepare, npm run build, etc., it all compiles fine, no errors, in terms of building, but it does build the math.d.ts file with the mention of Bezier without any require/import for it.

**Comment by ItzRabbs** [comment]
I'm getting this issue from simply using the latest 1.1.7 build in my package.json file.

I only have one import and that's `import { createCursor } from 'ghost-cursor';`

**Comment by Niek** [comment]
Can you check if `v1.1.8` that I just pushed to NPM resolves this?

If not, can you share some reproducible minimal sample code?

**Comment by ItzRabbs** [comment]
Here's a sample. I saved it down to `run.ts` and then I run `tsc run.ts`.

```
import * as Puppeteer from 'puppeteer';
import { createCursor } from 'ghost-cursor';

// tslint:disable-next-line: no-floating-promises
(async () => {
  const browser = await Puppeteer.launch({ headless: false });

  const page = await browser.newPage();
  await page.goto('https://www.google.com');
  await page.waitForNetworkIdle();

  const cursor = createCursor(page);
  await cursor.moveTo({ x: 5, y: 5 });
})();
```

**Comment by ItzRabbs** [comment]
Any luck with this?

**Comment by Niek** [comment]
I just published v1.1.9 with BezierJS 6 - can you check if this fixed the issue?

**Comment by Niek** [comment]
@ItzRabbs The BezierJS types are quite outdated, but I guess it doesn't really matter. I have published v1.1.10 now with the types added, can you check if this solves it?

**Comment by ItzRabbs** [comment]
Created a PR https://github.com/Xetera/ghost-cursor/pull/53

Hoping you can merge this and create a new release?

**Comment by Niek** [comment]
Released as 1.1.11 - please test: https://www.npmjs.com/package/ghost-cursor

**Comment by ItzRabbs** [comment]
Works well now. Thank you!
